umap/static/umap/js/modules/sync/updaters.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * This file contains the updaters: classes that are able to convert messages
+ * received from another party (or the server) to changes on the map.
+ */
+
+class BaseUpdater {
+  constructor(map) {
+    this.map = map
+  }
+
+  updateObjectValue(obj, key, value) {
```

Comment:
I had a look 👀 at how to put this code in the [https://github.com/yohanboniface/Leaflet.FormBuilder](Leaflet.Formbuilder] repository, and I think we might be better letting it here.

This is mainly because there are no unit tests (and no CI) there, unless I missed them. We can probably add tests and a CI there if we believe it is useful, and migrate the code there, but I also see benefits keeping the code here, as it's where it's small enough, and where it is used. Also, Its purpose seems a bit orthogonal to the FormBuilder one, and importing it from there seems a bit exotic.